File: TransCoder_tokenized_test_set_functions/python/K_MAXIMUM_SUM_COMBINATIONS_TWO_ARRAYS.py
def KMaxCombinations ( A, B, N, K ) :
    pq = PriorityQueue ( )
    for i in range ( 0, N ) :
        for j in range ( 0, N ) :
            a = A [ i ] + B [ j ]
            pq . put ( ( - a, a ) )
    count = 0
    while ( count < K ) :
        print ( pq . get ( ) [ 1 ] )
        count = count + 1
    

File: TransCoder_tokenized_test_set_functions/java/K_MAXIMUM_SUM_COMBINATIONS_TWO_ARRAYS.java
static void KMaxCombinations ( int A [ ], int B [ ], int N, int K ) {
  PriorityQueue < Integer > pq = new PriorityQueue < Integer > ( Collections . reverseOrder ( ) );
  for ( int i = 0;
  i < N;
  i ++ ) for ( int j = 0;
  j < N;
  j ++ ) pq . add ( A [ i ] + B [ j ] );
  int count = 0;
  while ( count < K ) {
    System . out . println ( pq . peek ( ) );
    pq . remove ( );
    count ++;
  }
}